sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -537,7 +543,7 @@ export interface ImportCertificateOptions extends coreHttp.OperationOptions {
   /**
    * The management policy.
    */
-  policy?: CertificatePolicy;
+  policy?: ImportCertificatePolicy;
```

Comment:
Breaking change concern! 

I don't know TS enough to verify whether this is or isn't a breaking change! The equivalent inline here would be:

```
---    policy?: CertificatePolicyProperties & RequireAtLeastOne<PolicySubjectProperties>;
+++ policy?: CertificatePolicyProperties & Partial<PolicySubjectProperties>;
```

Removing PolicySubjectProperties would be a breaking change, but in this case we're loosening restrictions from: I require either subject or SAN to I make them all optional (but you can still use the same set of fields)

Would someone be kind enough to validate whether this is a breaking change?

Reply:
Loosening restrictions is an additive, change, right? 🌞 I think it's fine.